recipes/notolog/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "notolog" %}
+{% set version = "0.9.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/notolog-{{ version }}.tar.gz
+  sha256: ab61b56d882e8b888b676a08a6fefd8dc222ed48160eb57f448dec5a8f95b82c
+
+build:
+  entry_points:
+    - notolog=notolog.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - pip
+  run:
+    - python >=3.9,<4.0
+    - cffi ==1.16.0
+    - click ==8.1.7
+    - cryptography ==42.0.7
+    - emoji ==2.12.1
+    - iniconfig ==2.0.0
+    - Markdown ==3.6
+    - packaging ==24.0
+    - pluggy ==1.5.0
+    - pycparser ==2.22
+    - Pygments ==2.18.0
+    - PySide6 ==6.7.0
+    - qasync ==0.27.1
+    - tomli ==2.0.1
+    - tomli-w ==1.0.0
+    - typing_extensions ==4.11.0
```

Comment:
These very strict dependencies will make it very difficult in the future to install your package because of conflicts. It does not prevent a merge, but you might want to consider relaxing these in the future.

Reply:
I'll take a look at this in the next version of the app, thank you.